OneBusAway/ui/trip_details/VehicleMapController.swift

```diff
@@ -119,21 +122,12 @@ class VehicleMapController: UIViewController, MKMapViewDelegate {
         self.mapView.snp.makeConstraints { (make) in
             make.edges.equalToSuperview()
         }
-
-        let blurContainer = OBAVibrantBlurContainerView.init(frame: CGRect.zero)
-        self.view.addSubview(blurContainer)
-        blurContainer.snp.makeConstraints { (make) in
-            make.width.equalTo(40)
-            make.height.equalTo(30)
-            make.right.bottom.equalToSuperview().offset(-OBATheme.defaultPadding)
-        }
-
-        blurContainer.vibrancyEffectView.addSubview(self.toggleButton)
-        self.toggleButton.snp.makeConstraints { make in
-            make.edges.equalToSuperview()
-        }
-        self.toggleButton.addTarget(self, action: #selector(toggleButtonTapped), for: .touchUpInside)
-        self.toggleButton.isSelected = self.expanded
+        
+        
+        //Moved toggle and location hover bars into setup functions
+        self.createToggleHoverBar()
```

Comment:
Since the toggle is a blur container view, i think it's separate from the hover bar. Sorry i misnamed the toggle setup function.